config/samples/cluster_v1_cri.yaml

```diff
@@ -409,6 +422,25 @@ spec:
           name: yql-agent-logs
         spec: *logsVolumeSpec
 
+  # queueAgents:
+  #   image: ghcr.io/ytsaurus/ytsaurus:stable-24.1.0-relwithdebinfo
+
+  #   instanceCount: 1
+
+  #   loggers: *loggers
+
+  #   locations:
+  #     - locationType: Logs
+  #       path: /yt/queue-agent-logs
+
+  #   volumeMounts:
+  #     - name: queue-agent-logs
+  #       mountPath: /yt/queue-agent-logs
+
+  #   volumeClaimTemplates:
+  #     - metadata:
+  #         name: queue-agent-logs
+  #       spec: *logsVolumeSpec
 
   queryTrackers:
```

Comment:
I think you can add a separate sample for timbertruck/persistence uploader example since this one is about cri I think and it wouldn't be helpful for user to mix unrelated configurations in same example